packages/amplify-category-geo/src/service-utils/deviceLocationTrackingUtils.ts

```diff
@@ -49,6 +55,44 @@ export const createDeviceLocationTrackingResource = async (
   printer.success('Created device location tracking resources');
 };
 
+/**
+ * modify amplify device tracking resource
+ */
+export const modifyDeviceLocationTrackingResource = async (
+  context: $TSContext,
+  parameters: DeviceLocationTrackingParameters,
+): Promise<void> => {
+  const authResourceName = await getAuthResourceName(context);
+  // generate CFN files
+  const templateMappings = await getTemplateMappings(context);
+  const deviceLocationTrackingStack = new DeviceLocationTrackingStack(
+    new App(),
+    'DeviceTrackingStack',
+    { ...parameters, ...templateMappings, authResourceName },
+  );
+  generateTemplateFile(deviceLocationTrackingStack, parameters.name);
+  saveCFNParameters(parameters);
+  stateManager.setResourceInputsJson(
+    pathManager.findProjectRoot(),
```

Comment:
stateManager calls implicitly call `pathManager.findProjectRoot()` if undefined is passed in as the first arg. If we remove this call, then `pathManager` does not need to be imported into this file.

```suggestion
    undefined,
```